src/is_hiragana.rs

```diff
@@ -11,10 +11,7 @@
 use crate::utils::is_char_hiragana::*;
 
 pub fn is_hiragana(input: &str) -> bool {
-    if input.is_empty() {
-        return false;
-    }
-    input.chars().all(is_char_hiragana)
+    !input.is_empty() && input.chars().all(is_char_hiragana)
```

Comment:
I think the previous version is more readable

Reply:
Imo this is cleaner as the function represents a single bool expression that is written there. But I actually didn't intend to put this into this PR.

Reply:
The single bool expression is the reason why it's less readable.
What happens on empty input? -> need to look at the whole bool expression instead of one line

What happens on other input? -> need to look at the whole bool expression instead of one line

Clean is quite subjective, but according to the "Clean Code" book this is discouraged (The Stepdown Rule)

Reply:
Yeah its indeed pretty subjective and as I didn't intend to PR this part anyways I'm fine with keeping it the way it was. I just changed it while going through the code when adjusting the lib for my needs.

Reply:
can you update the PR? LGTM to me, except the bool expression